eliot-service/eliot/app.py

```diff
@@ -56,67 +61,55 @@ def build_config_manager():
-    )
-    required_config.add_option(
-        "statsd_host", default="localhost", doc="Hostname for statsd server."
-    )
-    required_config.add_option(
-        "statsd_port", default="8124", doc="Port for statsd server.", parser=int
-    )
-    required_config.add_option(
-        "statsd_namespace", default="", doc="Namespace for statsd metrics."
-    )
-    required_config.add_option(
-        "secret_sentry_dsn",
-        default="",
-        doc=(
-            "Sentry DSN to use. If this is not set an unhandled exception logging "
-            "middleware will be used instead. "
-            "See https://docs.sentry.io/quickstart/#configure-the-dsn for details."
-        ),
-    )
-    required_config.add_option(
-        "symbols_cache_dir",
-        default="/tmp/cache",
-        doc="Location for caching symcache files.",
-    )
-    required_config.add_option(
-        "symbols_urls",
-        default="https://symbols.mozilla.org/try/",
-        doc="Comma-separated list of urls to pull symbols files from.",
-        parser=ListOf(str),
-    )
+    class Config:
+        local_dev_env = Option(
+            default="False",
+            parser=bool,
+            doc="Whether or not this is a local development environment.",
+            alternate_keys=["root:local_dev_env"],
+        )
+        host_id = Option(
+            default="",
+            doc=(
+                "Identifier for the host that is running Eliot. This identifies "
+                "this Eliot instance in the logs and makes it easier to correlate "
+                "Eliot logs with other data. For example, the value could be a "
+                "public hostname, an instance id, or something like that. If you do not "
+                "set this, then socket.gethostname() is used instead."
+            ),
+            alternate_keys=["root:host_id"],
+        )
+        logging_level = Option(
+            default="INFO",
+            doc="The logging level to use. DEBUG, INFO, WARNING, ERROR or CRITICAL",
+        )
+        statsd_host = Option(default="localhost", doc="Hostname for statsd server.")
+        statsd_port = Option(default="8124", doc="Port for statsd server.", parser=int)
+        statsd_namespace = Option(default="", doc="Namespace for statsd metrics.")
+        secret_sentry_dsn = Option(
+            default="",
+            doc=(
+                "Sentry DSN to use. If this is not set an unhandled exception logging "
+                "middleware will be used instead. "
+                "See https://docs.sentry.io/quickstart/#configure-the-dsn for details."
+            ),
+        )
+        symbols_cache_dir = Option(
+            default="/tmp/cache",
+            doc="Location for caching symcache files.",
+        )
+        symbols_urls = Option(
+            default="https://symbols.mozilla.org/try/",
+            doc="Comma-separated list of urls to pull symbols files from.",
+            parser=ListOf(str),
+        )
```

Comment:
Note that the configuration is the same, so we don't have to make any changes to infra.